x-pack/legacy/plugins/lens/public/indexpattern_plugin/field_item.tsx

```diff
@@ -5,15 +5,18 @@
  */
 
 import React from 'react';
-import { IndexPatternField, DraggedField } from './indexpattern';
+// @ts-ignore
+import { fieldFormats } from '../../../../../../src/legacy/ui/public/registry/field_formats';
```

Comment:
Will continue reviewing next week, just a quick question, why do you need fieldFormats here?

Reply:
Good catch- I created this PR by splitting up a larger chunk of work into 2 pieces, and this is the second piece. In a follow-up PR we will have a popover to show histograms/terms of sampled field data, and that needs to be formatted